examples/demo-atmos-cli-imports/configs.d/vendor.yaml

```
@@ -0,0 +1,9 @@
+vendor:  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)